homeassistant/components/trafikverket_weatherstation/const.py

```diff
@@ -7,3 +7,13 @@
 ATTRIBUTION = "Data provided by Trafikverket"
 ATTR_MEASURE_TIME = "measure_time"
 ATTR_ACTIVE = "active"
+
+NONE_IS_ZERO_SENSORS = {
```

Comment:
Since this is specific to sensors, maybe keep this inside `sensor.py`?

Reply:
I think it makes most sense to keep it in the const file with all others. There is only sensor platform in this integration.